### PR TITLE
NAS-114915 / 22.12 / Option to global set SED password missing from Advanced

### PR DESCRIPTION
**Pull request by AlexKarpov98**
Testing:

System -> Advanced
Screenshot_22

### CODE REVIEW

File: src/app/pages/system/advanced/advanced-settings.component.ts [hunk 1]
```diff
@@ -490,6 +506,9 @@ export class AdvancedSettingsComponent implements OnInit {
     if ([AdvancedCardId.Kernel].includes(name)) {
```
Comment (undsoft) [inline src/app/pages/system/advanced/advanced-settings.component.ts:506]: Or just add an array element here.

**Review by undsoft — CHANGES_REQUESTED** [review]
1. Ticket's fix version is set to SCALE-22.02.1. Please retarget PR to stable/angelfish.
2. Shouldn't there be some indication when password is set?

**Comment by AlexKarpov98** [comment]
Targeted to stable/angelfish https://github.com/truenas/webui/pull/6454

**Comment by AlexKarpov98** [comment]
> 2. Shouldn't there be some indication when password is set?

How can confirm if a password was set?
Advanced Config does not let to know it.

**Pull request closed without merging**